inst/rmarkdown/templates/josis/skeleton/josisexample.bib

```diff
@@ -0,0 +1,94 @@
+%% This BibTeX bibliography file was created using BibDesk.
+%% http://bibdesk.sourceforge.net/
+
```

Comment:
Are you using all the citation in this file for the skeleton ? 

If not, it would be interesting to reduce the example bib file we include.